unraid_mcp/tools/system.py

```
@@ -197,6 +207,107 @@ def analyze_disk_health(disks: list[dict[str, Any]], disk_type: str) -> dict[str
+      }
+    }
+    """
+    try:
+        logger.info("Executing get_metrics")
+        response_data = await make_graphql_request(query)
+        raw_metrics = response_data.get("metrics", {})
+        if not raw_metrics:
+            raise ToolError("No metrics returned from Unraid API")
+
+        # Format bytes to human-readable
+        def format_bytes(b: Any) -> str:
```

Ruff (0.14.7)

Dynamically typed expressions (typing.Any) are disallowed in `b`

(ANN401)